### ISSUE TITLE
Why was `Builder().textLength()` removed in 1.0.3

### ISSUE THREAD

**Issue by shahimclt** [iamageo/AnotherReadMoreLibrary#40]
the readme has the following snippet: 

```
val anotherReadMore: AnotherReadMore = AnotherReadMore.Builder(this)
    .textLength(100, AnotherReadMore.TYPE_LINE)
    .moreLabel("mais")
    .lessLabel("menos")
    .build()
```

but I see that the `textLength` method has been removed in v1.0.3. Any particular reason for that? Or is there an alternative method to set the max lines?

**Comment by iamageo** [comment]
Sorry, I was refactoring the library code and I must have removed it without seeing it, I will release a new version with the indication of ownership again.

**Comment by shahimclt** [comment]
Thank you

**Comment by iamageo** [comment]
implemented in version [1.0.6](https://github.com/iamageo/AnotherReadMoreLibrary/releases/tag/1.0.6)
